advanced-usage/index.md

```diff
@@ -0,0 +1,286 @@
+- Regular widgets can be configured by clicking the three dots in the top right corner of the widget
+- Mini widgets that support configuration can be clicked on anywhere on the widget to open the configuration options
+    - This is currently only possible during operation, *not* while in edit mode
+
+### Widgets
+
+There are several types of widgets available, and in future it will be possible to create, import, and use custom widgets as well.
+
+#### Regular Widgets
+
+##### Attitude HUD
+
+The attitude HUD widget displays the vehicle's pitch and roll as a heads-up display overlay:
+{{ easy_image(src="attitude-hud-widget", width=300, center=true) }}
+
+It is possible to configure which components get displayed, as well as the line colour:
+{{ easy_image(src="attitude-hud-config", width=500, center=true) }}
+
+##### Compass
+
+The compass widget displays the vehicle's orientation as though looking at a compass in your hand:
+{{ easy_image(src="compass-widget", width=300, center=true) }}
+
+It is most useful for guided and/or autonomous control, where the main display is of the vehicle's position.
+
+It is possible to configure the vertical direction to be fixed to North or to the vehicle's forwards direction.
+{{ easy_image(src="compass-config", width=300, center=true) }}
+
+##### Compass HUD
+
+The compass HUD is a first-person compass view, as though inside a compass and looking in the direction the
+vehicle is pointing (its heading):
+{{ easy_image(src="compass-hud-widget", width=600, center=true) }}
+
+It is possible to configure whether the exact heading angle is shown, and the colour of the lines:
+{{ easy_image(src="compass-hud-config", width=500, center=true) }}
+
+##### Depth HUD
+
+The depth HUD indicates the vehicle's current depth as determined by its external pressure sensor:
+{{ easy_image(src="depth-hud-widget", width=200, center=true) }}
+
+This is primarily useful for underwater vehicles.
+
+Configuration determines whether the exact depth value is shown, and the colour of the lines:
+{{ easy_image(src="depth-hud-config", width=500, center=true) }}
+
+##### Indicators
+
+The indicators widget shows a collection of telemetry values, including the vehicle's attitude, GPS location,
+and optionally its power supply and usage and some debugging data: 
+{{ easy_image(src="indicators-widget", width=200, center=true) }}
+{{ easy_image(src="indicators-config", width=300, center=true) }}
+
+##### Image Viewer
+
+The image viewer widget shows an image that is accessible to the control station computer via its network.
+{{ easy_image(src="image-viewer-config", width=400, center=true) }}
+
+Images from the internet can be included (e.g. a logo for branding) as long as the computer has internet
+access when Cockpit is started.
+
+This is most useful for images hosted on the local network, and was designed to display the output of a
+self-replacing mjpeg like from an ESP32-Cam. It could also display images hosted by a
```

Comment:
I think it's worth talking to @Williangalvani to document that setup with the esp32-cam, as it's a cheap and readily available alternative.

Reply:
Fair enough, although that likely belongs somewhere other than in the Cockpit docs, even if just in a forum "build" thread that we can link to.